signac_dashboard/modules/document_list.py

```diff
@@ -22,17 +23,23 @@ class DocumentList(Module):
 
     def __init__(
         self,
-        name="Job Document",
         context="JobContext",
+        name=None,
         template="cards/document_list.html",
         max_chars=None,
         **kwargs,
     ):
+        # set name based on intialized context
+        if name is None:
+            if context == "JobContext":
+                name = "Job Document"
+            elif context == "ProjectContext":
+                name = "Project Document"
         super().__init__(name=name, context=context, template=template, **kwargs)
         self.max_chars = max_chars
 
-    def get_cards(self, job):
-        doc = OrderedDict(sorted(job.document.items(), key=lambda t: t[0]))
+    def get_cards(self, jobOrProject):
```

Comment:
Snake case:
```suggestion
    def get_cards(self, job_or_project):
```
Or possibly a broader name?
```suggestion
    def get_cards(self, obj):
```
We also need to change the base class so it takes a parameter here -- methods of the subclasses should have the same signature as the parent class's method whenever possible. https://github.com/glotzerlab/signac-dashboard/blob/9b7c59d4f8c7abab57ee723135ffc2d27c1cb586/signac_dashboard/module.py#L73

Reply:
I like keeping the descriptive name in the subclass method.

Would `get_cards` ever take some other kind of argument?

Reply:
It could take a list of jobs, like an aggregate. That's the other context that I had in mind. Then it'd be something like `job_or_jobs_or_project`...